packages/flutter_form_builder/lib/src/form_builder.dart

```diff
@@ -63,6 +63,9 @@ class FormBuilder extends StatefulWidget {
   /// Whether the form should auto focus on the first field that fails validation.
   final bool autoFocusOnValidationFailure;
 
+  /// Whether the form should scroll to the first field that fails validation.
+  final bool shouldScrollToInvalidField;
```

Comment:
Maybe can rename this property to `autoScrollOnValidationFailure` or `autoScrollToInvalidField`. What do you think?

Reply:
I prefer `autoScrollToInvalidField` actually. Will change it out.